src/lambda_streams/Animals.java

```diff
@@ -1,62 +1,173 @@
 package lambda_streams;
 
 import java.util.*;
+import java.util.stream.Collectors;
 
-class Animals {
+class Animals
+{
     //This class will be using streams.
     static List<String> animals = Arrays.asList("peacoCK","rabbit","chiwawa","OranguTAN","vipeR","cobra","paNDa","bUffalo","DeeR","maLLard");
 
-    public static void main(String[] args) {
+    public static void main(String[] args)
+    {
         System.out.println("original animals : " + animals);
 
-        //clean up the animals array by using the capsFirst() method. follow instructions in the method definition. 
+        //clean up the animals array by using the capsFirst() method. follow instructions in the method definition.
         List<String> cleaned = capsFirst(animals, false);
-        System.out.println(cleaned);
-       
+        System.out.println("Cleaned: " + cleaned);
+
         //do not modify these addAnimal() method invocations
         addAnimal("rEIndeeR");
         addAnimal("Platypus");
         addAnimal("frOg");
         addAnimal("lEOpArD");
+        System.out.println("Add animals: " + animals);
         //---------------------------------------
 
         capsFirst(animals,true);
-        System.out.println(animals);
+        System.out.println("Caps first: " + animals);
 
         List<String> lowered = lowerFirst(animals,false);
-        System.out.println(lowered);
+        System.out.println("Lowered first: " + lowered);
+
+        List<String> flip = flipAnimals(false);
+        System.out.println("Flip animals: " + flip);
+
+        List<String> sort = sortAnimals(false);
+        System.out.println("Sorted animals: " + sort);
+
+
+
+        System.out.println(" ");
+        System.out.println("Begin Lambdas");
+
+        LambdaListBool val1 = (list, mutate) ->
+        {
+            flipAnimals(mutate);
+            return list;
+        };
+        System.out.print("Lambda flipAnimals: ");
+        System.out.println(val1.calc(animals, true));
```

Comment:
The expectation was for the methods below to be refactored so they accept a 'lambda' argument -- the provided lambda was then to be used to fulfill the requirements of the method (i.e. the method signature of 'flipAnimals(boolean)' would become 'flipAnimals(boolean, lambda)' ).